guides/index.md

```diff
@@ -1,3 +1,39 @@
-This documentation is a work in progress and is supposed to be a very high level overview of the Cardstack architecture and how to use various parts of the system. These pages are not designed to be very detailed **API** documentation and may have some inconsistencies at first.
 
-There should be a pencil icon at the top right of every page that will allow you to edit this documentation so please feel free to contribute if you know how to make things a bit better!
+These work-in-progress documentation resources provide a high level overview of features, the project architecture, how to use Cardstack to power your own creations, and some pointers for developers who are working on the framework itself.
+
+## What is Cardstack?
+
+Cardstack is an open source framework that enables Web 3.0. “Cards” form the building blocks that allow for bringing in data from disparate sources, beautiful UI animations, encapsulated and flexible data schemas, and a quick ramp-up experience. Both JavaScript developers and end-users are empowered to add common application features (CMS, notifications, authentication).
+
+Cardstack is a full-stack development environment, meaning that it spans everything from the front-end browser experience, to API layers, to the database and caching. Under the hood, it makes use of other libraries such as Node, Koa, Ember.js, and postgres, but someone does not need to know those tools in order to build something of their own using Cardstack.
+
+## Why use Cardstack?
+
+To understand what Cardstack can do, and why we think of it as the next generation web experience, it is helpful to first take a look at the status quo for the web. Most serious applications have to tie together many different data sources into a unified experience, and it is a struggle. A traditional web app focuses first on what goes _inside_ a widget, and adds nonstandard code around the edges to patch everything together. There's major friction between the front end, back end, database, and third party APIs. Once you string it all together, and try to make changes, everything breaks.
+
```

Comment:
> There's major friction between the front end, back end, database, and third party APIs.

I believe the friction is with the schema right? So in traditional apps, the schema is defined at the database level, API level, and front-end. With Cardstack, you only define the schema once, and the hub will take care of indexing, serving JSON to the front end, and generating the ember data models, so everything stays in sync.

If this is correct, we should mention it

Reply:
That matches my understanding! That's a great way to put it. Added.